plugin/src/main/scala/io/github/retronym/dependencyanalyzer/DependencyAnalyzer.scala

```diff
@@ -0,0 +1,100 @@
+package plugin.src.main.scala.io.github.retronym.dependencyanalyzer
```

Comment:
should we use a different package name when we are doing this? It seems weird to have a copy of the code without putting it in a new package.